src/Illuminate/Database/Grammar.php

```diff
@@ -116,6 +116,62 @@ protected function wrapValue($value)
         return $value;
     }
 
+    /**
+     * Convert array into sql group.
+     *
+     * @param  mixed  $values
+     * @return string
+     */
+    public function group($values)
```

Comment:
I would call it `tuple`, as grouping in SQL can have different meaning. And therefore change the other methods to `tuplify` etc.


Mainly to avoid confusion with `groupBy`.

Reply:
Agree